README.md

```diff
@@ -157,6 +166,10 @@ Short description.
 
   App that calculates the climate impact of your daily choices by connecting to apps and services you already use: your personal and automated carbon tracker.
 
+- [Lentera Bumi Nusantara](http://www.lenterabumi.com/aboutus.html/) | Indonesia | 2015
```

Comment:
I tried to check their website but it seems like their domain expired, so there's nothing there. I would suggest updating the URL if they moved somewhere else, or removing this entry if they are not active anymore.